index.html

```diff
@@ -12,12 +12,14 @@
           <a class="sidebar-button home-button" href="index.html">home</a>
           <a class="sidebar-button map-button" href="/map/map.html">map</a>
           <a class="sidebar-button data-button" href="/data/data.html">data</a>
+          <a class="sidebar-button us-button" href="/us/us.html">us</a>
```

Comment:
If you add a link to the sidebar remember to change it for all the pages, including the `template.html`.